### PR TITLE
Global and page styles

### PR DESCRIPTION
**Pull request by tinydinosaurs**
I caused some weird problems and I'm not sure how. I hope this fixes it.

### CODE REVIEW

File: public/styles/_pages.scss [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
-	border: 10px solid #ccc;
+@import 'variables';
+
+img.details-thumb {
+max-width: 100%;
 }
+
+.stories-details {
+	overflow: hidden;
+>>>>>>> Stashed changes
```
Comment (alarner) [inline public/styles/_pages.scss]: Remove this merge stuffs

**Comment by NJHawthorne** [comment]
👍

**Comment by hammour** [comment]
👍

**Comment by ihatetoast** [comment]
👍

**Pull request merged**